pkg/cli/admin/inspect/inspect.go

```diff
@@ -108,7 +108,7 @@ func NewCmdInspect(streams genericiooptions.IOStreams) *cobra.Command {
 		Run: func(c *cobra.Command, args []string) {
 			kcmdutil.CheckErr(o.Complete(args))
 			kcmdutil.CheckErr(o.Validate())
-			kcmdutil.CheckErr(o.Run())
+			kcmdutil.CheckErr(o.Run(context.TODO()))
```

Comment:
Run usually does not get any context. It is better to define context.TODO inside `Run`